osutil/kcmdline.go

```diff
@@ -175,11 +175,7 @@ func KernelCommandLineSplit(s string) (out []string, err error) {
 	return out, nil
 }
 
-// KernelCommandLineKeyValues returns a map of the specified keys to the values
-// set for them in the kernel command line (eg. panic=-1). If the value is
-// missing from the kernel command line or it has no value (eg. quiet), the key
-// is omitted from the returned map.
-func KernelCommandLineKeyValues(keys ...string) (map[string]string, error) {
+func kernelCommandLineKeyValues(omitNoVal bool, keys ...string) (map[string]string, error) {
```

Comment:
as we are touching this, does it make sense to move it to use ParseKernelCommandline instead of *Split ?